<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGuiWxSetting</class>
 <widget class="QDialog" name="QGuiWxSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录入设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_aliasset">
      <attribute name="title">
       <string>别名设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QTableWidget" name="tableWidget_alias">
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>实名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>别名</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="pushButton_alias_add">
           <property name="text">
            <string>新增</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_alias_del">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_alias_apply">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_nomean">
      <attribute name="title">
       <string>无义字符</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QTextEdit" name="textEdit_nomean_showedit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLineEdit" name="lineEdit_nomean_lineedit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_nomean_add">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_nomean_del">
           <property name="text">
            <string>移除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nomean_apply">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_replace">
      <attribute name="title">
       <string>替换字符</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QTableWidget" name="tableWidget_replace">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>原字符</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>替换为</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QPushButton" name="pushButton_replace_add">
           <property name="text">
            <string>新增</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_replace_del">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_alias_del</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setalias_del()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>-103</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_alias_apply</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setalias_apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>-110</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_nomean_add</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setnomean_add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>-125</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_nomean_del</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setnomean_del()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>-26</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_nomean_apply</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setnomean_apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>-25</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_replace_add</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setreplace_add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>-27</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_replace_del</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setreplace_del()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>-36</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setreplace_apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>-51</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_alias_add</sender>
   <signal>clicked()</signal>
   <receiver>QGuiWxSetting</receiver>
   <slot>on_setalias_add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>-682</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_setalias_add()</slot>
  <slot>on_setalias_del()</slot>
  <slot>on_setalias_apply()</slot>
  <slot>on_setnomean_add()</slot>
  <slot>on_setnomean_del()</slot>
  <slot>on_setnomean_apply()</slot>
  <slot>on_setreplace_add()</slot>
  <slot>on_setreplace_del()</slot>
  <slot>on_setreplace_apply()</slot>
 </slots>
</ui>
